adviser/base/argo-workflows/qeb-hwt-integration.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: thamos-advise
+  annotations:
```

Comment:
annotations and labels should be removed, they are managed via the kustomization